go.mod

```diff
@@ -252,3 +252,5 @@ require (
 )
 
 go 1.22
+
+replace github.com/ipfs/boxo => github.com/MichaelMure/boxo v0.0.0-20240730133910-2b6f29f9b7f9
```

Comment:
- [ ] Remove and rebase once https://github.com/ipfs/boxo/pull/630 / https://github.com/ipfs/kubo/pull/10462 land